### ISSUE TITLE
UI not updating bets

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#1283]
Find and fix the problem Chuck was having:

> Chuck: Cliff: check 388634. I called (and it shows it) but the amount in front of me stayed at 400.
Chuck: Similar problem with 388641. Dan bets, but it doesn't show in front of him.
Chuck: Cliff, I'm consistently not seeing the amount bet. And when I click "pot" in the betting box it doesn't update. Not sure if it's just me?
gerdog: i'm not having that issue

**Comment by ctm** [comment]
In hand 388634, Chuck was the small blind for 400, LingCod was the big blind for 800, smalltalkdan folded, gerdog raised 800 to 1,600, LfromHell called and then Chuck called. I'm guessing from what he wrote in the lobby, that when he called, instead of the amount in front of him showing 1,600, it only showed 400.

This is where it would be real handy to have the ability to play back a hand, message by message, through the UI so that I could see what the UI would show. That wouldn't _necessarily_ reproduce this&mdash;or any other&mdash;bug, because bugs are, by definition, something going wrong, but in many cases, and _perhaps_ this one, it would. If I had that tool, it's the first I'd reach for.

I can, however, use a `force-status` build and get similar messages to what Chuck probably received to run through the interface. Setting up `force-status` is a little time consuming, but I can do it today.

My guess is neither `force-status` nor a hypothetical replay-the-messages-from-the-database would reproduce this issue, because nobody else has reported it, but I'll know later today, and this bug report pushes me that much closer to taking time to build the repay-the-messages tool.

```
mb2=> select id, player_id, received_at at time zone 'america/denver', message from public_table_messages where hand_id = 388634 order by received_at;
    id    | player_id |          timezone          |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    message                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
----------+-----------+----------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 10341160 |           | 2024-01-15 18:51:29.208451 | {"Dealing": ["400 800 No-Limit Texas Hold’em (High only)", 0, "NoLimit", null, false, 388634, true, false, 0]}
 10341188 |           | 2024-01-15 18:51:31.118652 | {"BigBlindAntes": [18, 800, false]}
 10341189 |           | 2024-01-15 18:51:31.12345  | {"Blinds": [3, 400, false, "Small"]}
 10341190 |           | 2024-01-15 18:51:31.127828 | {"Blinds": [18, 800, false, "Big"]}
 10341191 |           | 2024-01-15 18:51:31.132065 | {"BeginRound": [0, null, 800, 0, false, 153]}
 10341192 |           | 2024-01-15 18:51:31.136925 | {"Status": {"board": {"grid": null, "cards": {"cards": []}}, "button": {"seat": 2, "small_blind": 4}, "players": [{"id": 2, "seat": 2, "cards": {"cards": ["UnknownDown", "UnknownDown"]}, "chips": 30000, "action": 0, "status": "Live", "lammers": null, "pot_size": null, "vacation": false, "pot_number": null, "n_rebuys_remaining": null}, {"id": 3, "seat": 4, "cards": {"cards": ["UnknownDown", "UnknownDown"]}, "chips": 32750, "action": 400, "status": "Live", "lammers": null, "pot_size": null, "vacation": false, "pot_number": null, "n_rebuys_remaining": 0}, {"id": 18, "seat": 5, "cards": {"cards": ["UnknownDown", "UnknownDown"]}, "chips": 7650, "action": 800, "status": "Live", "lammers": null, "pot_size": null, "vacation": false, "pot_number": null, "n_rebuys_remaining": null}, {"id": 6, "seat": 7, "cards": {"cards": ["UnknownDown", "UnknownDown"]}, "chips": 64961, "action": 0, "status": "Live", "lammers": null, "pot_size": null, "vacation": false, "pot_number": null, "n_rebuys_remaining": 0}, {"id": 11, "seat": 8, "cards": {"cards": ["UnknownDown", "UnknownDown"]}, "chips": 37900, "action": 0, "status": "Live", "lammers": null, "pot_size": null, "vacation": false, "pot_number": null, "n_rebuys_remaining": 0}], "solo_ante": {"seat": 5, "all_in": false, "amount": 800}, "action_type": "Betting", "next_to_act": 7, "cards_are_settable": false}}
 10341193 |           | 2024-01-15 18:51:31.142143 | {"Remind": [20000, 800, "2024-01-16T01:51:51.117473410Z", 6, 20]}
 10341200 |           | 2024-01-15 18:51:37.45835  | {"Folds": [6, false, null, null]}
 10341201 |           | 2024-01-15 18:51:37.46356  | {"Remind": [20000, 800, "2024-01-16T01:51:57.457051771Z", 11, 20]}
 10341204 |           | 2024-01-15 18:51:40.607134 | {"RaisesNtoM": [11, 800, 1600, null, false, 1600]}
 10341205 |           | 2024-01-15 18:51:40.612157 | {"Remind": [20000, 1600, "2024-01-16T01:52:00.605725035Z", 2, 20]}
 10341212 |           | 2024-01-15 18:51:47.79247  | {"Chat": [11, "L!!!"]}
 10341215 |           | 2024-01-15 18:51:51.374928 | {"Calls": [2, null, false, 1600]}
 10341216 |           | 2024-01-15 18:51:51.379878 | {"Remind": [20000, 1200, "2024-01-16T01:52:11.373526970Z", 3, 20]}
 10341229 |           | 2024-01-15 18:52:05.091046 | {"Calls": [3, null, false, 1600]}
```

**Comment by ctm** [comment]
BTW, I did set up `force-status` to reproduce the hand, but the bug didn't reproduce. I also found that Chuck closed the window and the new window he got didn't have the bug. That strongly suggests that even if I were to play back all of the messages directly from the database rather than an approximation via `force-status` that I wouldn't reproduce the bug.

Unfortunately, I was a bit under the weather today, so I never seriously thought about writing the playback tool (or doing important CPC business work, either).
